code/modules/modular_computers/file_system/programs/engineering/sm_monitor.dm

```diff
@@ -0,0 +1,137 @@
+/datum/computer_file/program/supermatter_monitor
+	filename = "smmonitor"
+	filedesc = "Supermatter Monitoring"
+	ui_header = "smmon_0.gif"
```

Comment:
Testing out the program, I did not see this icon update as crystals destabilized - I saw a white box with an x in it. Is this checking the right file?